src/components/core/pivotTable/tests/agGridDataSource.spec.ts

```diff
@@ -8,19 +8,22 @@ import { createIntlMock } from "../../../visualizations/utils/intlUtils";
 const intl = createIntlMock();
 
 describe("getGridDataSource", () => {
-    function createMockedDataSource() {
+    function createMockedDataSource(customGridApi?: () => any) {
```

Comment:
How about renaming `customGridApi` to `customGridApiProps` and use spread operator while creating mocked grid API? See below.